lib/src/theme/overlay_filter.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:ui';
+import 'package:macos_ui/macos_ui.dart';
+import 'package:macos_ui/src/library.dart';
+
+/// Applies a blur filter to its child to create a macOS-style "frosted glass"
+/// effect.
+class MacosOverlayFilter extends StatelessWidget {
+  /// Applies a blur filter to its child to create a macOS-style "frosted glass"
+  /// effect.
+  ///
+  /// Used mainly for the overlays that appear from various macOS-style widgets,
+  /// like the pull-down and pop-up buttons, or the search field.
+  const MacosOverlayFilter({
+    Key? key,
+    required this.child,
+    required this.borderRadius,
+    this.height,
+    this.color,
+  }) : super(key: key);
+
+  final Widget child;
+
+  final BorderRadius borderRadius;
+
+  final double? height;
+
+  final Color? color;
```

Comment:
Would be nice for these to be documented

Reply:
In documenting these, I found that the height property can be omitted. The dimensions of the child widget are handled by it anyway; this is meant to be used as a decoration wrapper.